packages/shared/src/utils/assert.ts

```diff
@@ -0,0 +1,3 @@
+export function assertUnreachable(_: never): never {
+  throw new Error('There are more values to handle.')
+}
```

Comment:
I think we should rename this file to assertUnreachable, delete the one that's in the backend and use this one instead since it's exactly the same

Reply:
good catch